bugwarrior/services/logseq.py

```diff
@@ -181,16 +181,16 @@ def get_formatted_title(self):
 
     # get a list of tags from the task content
     def get_tags_from_content(self):
-        # this includes #tagname, but ignores tags that are in the #[[tag name]] format
+        # pattern match for #[[multi word]] tags and #single word tags
+        # but ignore any non-tag use of the # character in URLs
+        # like http://example.com/page#test or in `#code`
+        # Regex Pattern: (?<=\s)#【.*】|(?<=\s)#\S+
```

Comment:
Makes sense to me -- this fixes the bug with the negative look-behind for a space and simplifies the code somewhat by consolidating into a single pattern.

Would it be possible to simplify it further by eliminating the duplication?
```suggestion
        # Regex Pattern: (?<=\s)(#【.*】|#\S+)
```

My regex is rusty so I may be forgetting how parenthesis work.

Reply:
Yes, this works.

One minor edit, I forgot to add the `?` for non-greedy inclusion in the regex sample in the comment `(?<=\s)(#【.*?】|#\S+)`